codemp/server/server.h

```diff
@@ -152,6 +154,7 @@ typedef struct client_s {
 	int				lastClientCommand;	// reliable client message sequence
 	char			lastClientCommandString[MAX_STRING_CHARS];
 	sharedEntity_t	*gentity;			// SV_GentityNum(clientnum)
```

Comment:
The entity this points to can be either in the QVM or Native, right? So there's no way to use it safely, I think this should be deleted, only `gentityMapper` can be used safe.

Reply:
fix in f81e89390cae67f7cb8ad33d98048919f6604e7f